README.md

```diff
@@ -7,6 +7,10 @@ with [Plaid Link](https://plaid.com/docs/link/)
 
 React 16.8+
 
+### Known limitations
+
+The selfie check feature in Plaid Identity Verification is not yet supported.
```

Comment:
Does this need an extra qualifier? I'm assuming if you run this in a browser, the whole "Go to this QR cpde on your phone to complete your selfie check" still works, right?